ror/spec/factories/comments.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+
+  factory :comment do
+    commenter     { 'factory comment commenter' }
+    body          { 'factory comment body' }
+    article_id    { 1 }
```

Comment:
FactoryBot can construct relationships.

```suggestion
    association :article
```

https://devhints.io/factory_bot#associations

Reply:
Fixed. Thanks for the link!